### PR TITLE
fr initial commit

### PR DESCRIPTION
**Pull request by Jean-Baptiste-Camps**
Here it is, not much for now. I'll add the remote model download.

### CODE REVIEW

**Comment by Jean-Baptiste-Camps** [comment]
@PonteIneptique

**Comment by Jean-Baptiste-Camps** [comment]
There are modifications to make to the code, such as not removing punctuation for instance.

File: pie_extended/models/fr/get.py [hunk 1]
```diff
@@ -9,16 +9,16 @@ def get_iterator_and_processor():
     tokenizer = FrMemorizingTokenizer()
     processor = FrRulesProcessor(
```
Comment (PonteIneptique) [inline pie_extended/models/fr/get.py]: Put directly MemoryzingProcessor here if Rules is not used

File: pie_extended/models/fr/processor.py [hunk 2]
```diff
@@ -7,24 +7,6 @@
 
 
 class FrRulesProcessor(RuleBasedProcessor):
```
Comment (PonteIneptique) [inline pie_extended/models/fr/processor.py]: Remove the declaration if not used.

File: pie_extended/models/fr/get.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+from .processor import FrRulesProcessor, FrGlueProcessor
+from pie_extended.pipeline.postprocessor.proto import RenamedTaskProcessor
+from .tokenizer import FrMemorizingTokenizer
+from pie_extended.pipeline.iterators.proto import DataIterator, GenericExcludePatterns
+from pie_extended.pipeline.postprocessor.memory import MemoryzingProcessor
+
+
+def get_iterator_and_processor():
```
Comment (PonteIneptique) [inline pie_extended/models/fr/get.py]: Should rename `fr.imports`

File: pie_extended/models/fr/__init__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+from ...utils import Metadata, File, get_path
+from .get import get_iterator_and_processor
```
Comment (PonteIneptique) [inline pie_extended/models/fr/__init__.py]: Remove this line when you have added the `imports.py` file

**Review by PonteIneptique — CHANGES_REQUESTED** [review]

**Pull request merged**